### PR TITLE
Add linter config for schemas

### PR DESCRIPTION
**Pull request by o-rumiantsev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +29,18 @@
...
+    "lint": "npm run lint:js && npm run lint:schema && npm run lint:json && npm run lint:md && npm run lint:yml",
+    "lint:js": "eslint --ignore-path .gitignore .",
+    "lint:schema": "eslint **/*.{category,domains} && prettier -c \"**/*.category\" \"**/*.domains\"",
+    "lint:json": "prettier -c \"**/*.json\" \"**/*.*rc\"",
+    "lint:md": "prettier -c \"**/*.md\"",
+    "lint:yml": "prettier -c \"**/*.yml\"",
+    "fmt": "npm run fmt:js && npm run fmt:schema && npm run fmt:json && npm run fmt:md && npm run fmt:yml",
+    "fmt:js": "prettier --write \"**/*.js\"",
+    "fmt:schema": "prettier --write \"**/*.action\" \"**/*.category\" \"**/*.domains\"",
+    "fmt:json": "prettier --write \"**/*.json\" \"**/*.*rc\"",
```
Comment (nechaido) [inline package.json]:
```suggestion
    "fmt:json": "prettier --write \"**/*.json\" \".*rc\"",
```

File: .prettierignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 dist/
-README.md
+.eslintignore
+.gitignore
+.prettierignore
+LICENSE
```
Comment (nechaido) [inline .prettierignore]: Add `package-lock.json`

File: package.json [hunk 3]
```diff
@@ -29,8 +29,8 @@
   ],
   "scripts": {
     "test": "npm run lint && metatests test/",
-    "lint": "eslint --ignore-path .gitignore . && prettier -c \"**/*.js\" \"**/*.json\" \"**/*.md\" \".*rc\" \"**/*.yml\"",
-    "fmt": "prettier --write \"**/*.js\" \"**/*.json\" \"**/*.md\" \".*rc\" \"**/*.yml\"",
+    "lint": "eslint \"**/*\" && prettier -c \"*\"",
+    "fix": "prettier --write \"*\" && eslint --fix \"**/*\"",
```
Comment (nechaido) [inline package.json]:
```suggestion
    "fmt": "prettier --write \"*\" && eslint --fix \"**/*\"",
```

File: test/schemas/.eslintrc.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+globals:
+  Validate: false
+  Enum: false
+  Flags: false
+  List: false
+rules:
+  camelcase: off
```
Comment (nechaido) [inline test/schemas/.eslintrc.yml]: Do we really need to add this rule?

File: test/schemas/modules/unsupported.ext [hunk 5]
```diff
@@ -1 +1 @@
-This file should not be loaded
+// This file should not be loaded
```
Comment (nechaido) [inline test/schemas/modules/unsupported.ext]: Is it really needed?

Comment (o-rumiantsev) [reply test/schemas/modules/unsupported.ext]: Yes, because linter fails while parsing this file

File: package.json [hunk 6]
```diff
@@ -29,8 +29,8 @@
   ],
   "scripts": {
     "test": "npm run lint && metatests test/",
-    "lint": "eslint --ignore-path .gitignore . && prettier -c \"**/*.js\" \"**/*.json\" \"**/*.md\" \".*rc\" \"**/*.yml\"",
-    "fmt": "prettier --write \"**/*.js\" \"**/*.json\" \"**/*.md\" \".*rc\" \"**/*.yml\"",
+    "lint": "eslint \"**/*\" && prettier -c \"*\"",
+    "fmt": "prettier --write \"*\" && eslint --fix \"**/*\"",
```
Comment (lundibundi) [inline package.json]: While we are at it, should we perhaps reorder them according to `lint` command (eslint first, then prettier)?

**Pull request merged**